### PR TITLE
Estate Planning name change, link name change

### PR DESCRIPTION
**Pull request by KishaT**
The content will need to be changed to reflect the new title. You can tag me to review when you're done. Thanks!

### CODE REVIEW

File: pages/estate-planning/index.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 
 styles:
 
 scripts:
 
-permalink: /estate-planning/
+permalink: /planning-for-life-events/
 ---
-# Estate planning
+# Planning for life events
```
Comment (line47) [inline pages/estate-planning/index.md]: We need to make some changes

**Comment by line47** [comment]
@JamelleWT good for me to review now?

**Comment by JamelleWT** [comment]
I wanted Kisha to give the language a second glance but yes, It'd updated,

Thanks,

File: pages/estate-planning/index.md [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
 
-Your estate is comprised of everything you own, including your TSP account. Making an estate plan ensures that your possessions are given to the people or organizations you care about most when you die. It’s a good idea to review your estate plan periodically, especially if your family or marital status changes. You may be changing jobs, getting married, or moving to another military base. All of these are great reasons to revisit your plan for your TSP account.
+You may be changing jobs, getting married, or moving to another military base. You may even be ready to retire. Making a plan for life changes includes what you want to happen with your TSP account. It's important to make a plan and revisit that plan periodically to make sure your wishes are honored.
```
Comment (PamGit2715) [inline pages/estate-planning/index.md]: Could we change this to read:
You may be changing jobs, getting married, moving to another military base, or getting ready for retirement. These are great reasons to make a plan for life changes, which includes plans for your TSP account. It's also important to revisit your plan periodically to ensure your wishes are honored.

**Pull request merged**